src/views/catalog/utils/quantity.ts

```diff
@@ -14,7 +14,7 @@ export const bytesFromQuantity = (
   quantity: string | number,
   precision = 0,
 ): [value: number, unit: string] => {
-  let byteSizeResult: ByteSizeResult = undefined;
+  let byteSizeResult: ByteSizeResult = { value: undefined, unit: undefined, long: undefined };
```

Comment:
Why have you added the `long` property? It doesn't look like it's used anywhere.

FYI, it's better to use `null` when initializing a variable yourself. I didn't see the original when I was working on it or I would have changed it.

Here's a good explanation of the difference:

Null: It is the intentional absence of the value. It is one of the primitive values of JavaScript.
Undefined: It means the value does not exist in the compiler. It is the global object.

Source: https://www.geeksforgeeks.org/undefined-vs-null-in-javascript/